transform.go

```diff
@@ -7,6 +7,8 @@ import (
 )
 
 const _defaultTitle = "Table of Contents"
+const _defaultTitleDepth = 1
+const _defaultMaxTitleDepth = 6
```

Comment:
naming: "default" usually means that it's the fallback value to something that can otherwise be changed. This is just the max title depth—it can't be changed.
I'll rename it—I have the code checked out already.